.circleci/config.yml

```diff
@@ -40,23 +31,48 @@ jobs:
       # auto build typescript definition files (not part of build as locally JsDoc alone is all we need)
       - run: npm run build:types
 
+      # persist built files and duplicate dependencies to workspace
+      - persist_to_workspace:
+          root: ~/
+          paths:
+              - repo/packages/*/node_modules
+              - repo/packages/*/dist
```

Comment:
should we add `- repo/node_modules` as well?

Reply:
We could do a couple of test builds to compare performance, but right now they are being added to the cache, which can persist longer (as long as yarn.lock doesn’t change) but can’t be targeted by glob syntax. Being `node_modules` can be rather large (yay, modern JS) I’ve seen the workspace persistence go slower than caching on that number of files, but not in a scientifically proven way. I’ll check it out more closely.